src/engine/Engine.ts

```diff
@@ -83,6 +101,14 @@ export interface IEngineOptions {
     * browsers or if there is a bug in excalibur preventing execution.
     */
    suppressMinimumBrowserFeatureDetection?: boolean;
+   
+   /*
+   *Specify how the game window is to be positioned when the DisplayMode.Position is chosen. This option MUST be specified
+   *if the DisplayMode is set as DisplayMode.Position. The position can be either a string or an AbsolutePosition. String must be in the
+   *format of css style background-position. 
+   */
+   
+   position?: string | IAbsolutePosition;
```

Comment:
Let's add a few examples to the documentation of valid positions, since diverse types are valid it may be good to give some examples.